apps/server/test/lexical/server/provider/handlers/go_to_definition_test.exs

```diff
@@ -61,11 +61,6 @@ defmodule Lexical.Server.Provider.Handlers.GoToDefinitionTest do
       {:ok, request} = build_request(uses_file_path, 4, 17)
 
       {:reply, %{result: %Location{} = location}} = handle(request, project)
-
-      assert location.range.start.line == 15
-      assert location.range.start.character == 7
-      assert location.range.end.line == 15
-      assert location.range.end.character == 12
```

Comment:
This was actually learned from `find_reference_test.exs`: https://github.com/lexical-lsp/lexical/blob/f8ef46b81b74b7a129ce17710f63826999b5b31a/apps/server/test/lexical/server/provider/handlers/find_references_test.exs#L70

Additionally, I feel that verifying these rows and columns doesn't really make much sense. It's sufficient to verify that the data structure is returned. 
Precise row and column verification has already been done in the `remote_control` app. Moreover, since these are transformed LSP rows and columns, they can't be directly used with the `decorate` function in `RangeSupport`.

Reply:
Yeah, you know i'm not in favor of those kinds of assertions.